src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylePairs = sourceString
+    .trim()
+    .split(';')
+    .map((pair) => pair.trim().split(':'))
+    .filter((pair) => pair.length > 1);
+
+  const styleObj = {};
+
+  stylePairs.forEach((pair) => {
+    const key = pair[0].trim();
+    const value = pair[1].trim();
```

Comment:
Trimming the key and value is good practice to ensure that any extra whitespace does not affect the resulting object.